bitnami/nginx-ingress-controller/templates/controller-hpa.yaml

```
@@ -1,5 +1,9 @@
 {{- if and .Values.autoscaling.enabled (eq .Values.kind "Deployment") }}
+{{- if semverCompare "<1.23-0" .Capabilities.KubeVersion.GitVersion }}
```

Could you please use the helper `common.capabilities.kubeVersion` instead of `.Capabilities.KubeVersion.GitVersion`?
```suggestion
{{- if semverCompare "<1.23-0" (include "common.capabilities.kubeVersion" .) }}
```

The helper takes under consideration the value `kubeVersion`, which allows users to generate templates targeted for a specific Kubernetes version.

https://github.com/bitnami/charts/blob/fbd75ab9a01efe8e52c696d2a137d9f13b8d564b/bitnami/common/templates/_capabilities.tpl#L3-L16